src/main/java/com/dumbdogdiner/stickyapi/bukkit/user/StickyUserBukkit.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2020 DumbDogDiner <dumbdogdiner.com>. All rights reserved.
+ * Licensed under the MIT license, see LICENSE for more information...
+ */
+package com.dumbdogdiner.stickyapi.bukkit.user;
+
+import com.dumbdogdiner.stickyapi.bukkit.item.generator.PlayerHeadGenerator;
+import com.dumbdogdiner.stickyapi.common.user.StickyUser;
+import org.bukkit.Bukkit;
+import org.bukkit.Location;
+import org.bukkit.OfflinePlayer;
+import org.bukkit.Sound;
+import org.bukkit.entity.Player;
+import org.bukkit.inventory.ItemStack;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.UUID;
+
+public class StickyUserBukkit extends StickyUser {
+    public StickyUserBukkit(Player p) {
+        super(p.getUniqueId(), p.getName());
+    }
+
+    public StickyUserBukkit(OfflinePlayer p) {
+        super(p.getUniqueId(), p.getName());
+    }
+
+    public StickyUserBukkit(StickyUser p){
+        super(p);
+    }
+
+    public StickyUserBukkit(UUID uniqueId){
+        super(uniqueId);
+    }
+
+    public boolean isOnline(){
+        return getAsOfflinePlayer().isOnline();
+    }
+
+    public boolean isBanned(){
+        return getAsOfflinePlayer().isBanned();
+    }
+
+    public boolean playSound(@NotNull Location location, @NotNull Sound sound, float v, float v1){
```

Comment:
```suggestion
    public boolean playSound(@NotNull Location location, @NotNull Sound sound, float v, float v1) {
```

Reply:
A more explicit variable name for `v` and `v1` would also be nice. I have no idea what those are.

Reply:
@NotZachery this is your code, I moved it here

Reply:
`v` is volume and `v1` is pitch according to Spigot's javadoc

Reply:
Doesn't matter who wrote it, a more explicit variable name would be preferable.

Reply:
fixed